typing/includemod_errorprinter.ml

```diff
@@ -709,7 +709,16 @@ let rec module_type ~expansion_token ~eqmode ~env ~before ~ctx diff =
       functor_params ~expansion_token ~env ~before ~ctx d
   | _ ->
```

Comment:
Could this pattern here be made exhaustive / non-fragile? It would help understand what is going on, in particular by making it easy to reason about which cases go to `dwith_context_and_elision` below.